frontend/coffee/profile.coffee

```diff
@@ -98,6 +98,60 @@ resetPasswordModalDialog = () ->
 
 $('#change-password').on('hidden.bs.modal', resetPasswordModalDialog)
 
+# Handling of the delete-account dialog
```

Comment:
Can we put this in a new `profile_view.coffee` instead (in anticipation of #420)? The current code in this file is just for the edit page.